.github/workflows/docker_ci.yml

```diff
@@ -1,41 +1,41 @@
-name: Docker Alpine MUSL CI
+name: Docker CI
 
 on:
   push:
     paths-ignore:
       - '**.md'
       - '**.yml'
-      - '!**/docker_alpine.yml'
+      - '!**/docker_ci.yml'
       - 'cmd/tools/**'
       - '!cmd/tools/builders/**.v'
   pull_request:
     paths-ignore:
       - '**.md'
       - '**.yml'
-      - '!**/docker_alpine.yml'
+      - '!**/docker_ci.yml'
       - 'cmd/tools/**'
       - '!cmd/tools/builders/**.v'
 
 concurrency:
   group: ${{ github.workflow }}-${{ github.ref == 'refs/heads/master' && github.sha || github.ref }}
   cancel-in-progress: true
 
+env:
+  V_CI_MUSL: 1
+
 jobs:
-  alpine-docker-musl-gcc:
+  alpine-musl-gcc:
```

Comment:
There is no possibility for confusion here, but imho `alpine-docker-musl` will fit better with `ubuntu-docker-musl` .